src/controllers/email-preview.js

```diff
@@ -0,0 +1,113 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import AppConstants from '../app-constants.js'
+
+import { notify } from './hibp.js'
+import { mainLayout } from '../views/main.js'
+import { emailPreview } from '../views/partials/email-preview.js'
+import { getTemplate, getPreviewTemplate } from '../views/email-2022.js'
+import { breachAlertEmailPartial } from '../views/partials/email-breach-alert.js'
+import { verifyPartial } from '../views/partials/email-verify.js'
+
+import { UnauthorizedError } from '../utils/error.js'
+import { getMessage } from '../utils/fluent.js'
+import { generateToken } from '../utils/csrf.js'
+import {
+  getNotifictionDummyData,
+  getVerificationDummyData,
+  sendEmail
+} from '../utils/email.js'
+
+const { EMAIL_RECIPIENT_DUMMY } = AppConstants
+
+function emailsPage (req, res) {
+  if (!AppConstants.EMAIL_PREVIEW_ENABLED) {
+    throw new UnauthorizedError(`${req.method} ${req.originalUrl}`)
```

Comment:
Since this isn't auth related, perhaps a 404 (not found error) would be more appropriate